pkg/pillar/cmd/msrv/msrv.go

```diff
@@ -629,6 +629,9 @@ func (msrv *Msrv) MakeMetadataHandler() http.Handler {
 
 		r.Post("/tpm/signer", msrv.handleSigner(&zedcloudCtx))
 
+		r.Post("/tpm/activatecredential/", msrv.handleActivateCredntial())
+		r.Get("/tpm/activatecredential/", msrv.handleActivateCredntial())
```

Comment:
Do you think it makes sense to separate msrv.handleActivateCredntial into two functions: one for post another for Get? chi will guarantee that r.Post will have only requests with POST method :)

Reply:
I'm not sure what would be the benefit, do you have something specific in mind?

Reply:
Just making function smaller :)

Reply:
I'd agree that two separate functions makes sensesince there is only one common line between the post and get parts of this function.

Reply:
done.